### ISSUE TITLE
Add support for Scion as an OrthogonalizedOptimizer subclass

### ISSUE THREAD

**Issue by mkhona-nvidia** [NVIDIA-NeMo/Emerging-Optimizers#23]
**Is your feature request related to a problem? Please describe.**
Scion (https://arxiv.org/abs/2502.07529) offers a Franke-Wolfe perspective and views (decoupled) weight decay as constrained optimization. It allows for transfer of hyperparameters across scale and is theoretically motivated. 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by skyw** [comment]
Addressed #70
